TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -92,17 +101,17 @@ class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
         }
     }
 
-    fun updateNote(title: String, text: String) {
-        // REVISIT: assuming that currently selected item is the note that just got updated
-        var note: Note?
+    fun updateNote(title: String) {
+        var inode: Inode?
 
         arrayAdapter?.let { array ->
-            note = array.getItem(selectedItemPosition)
+            inode = array.getItem(selectedItemPosition)
 
-            note?.let {
+            inode?.let {
+                DataProvider.notes.remove(it)
                 it.title = title
-                it.text = text
```

Comment:
why delete the text field update?

Reply:
jk i see that you now use "updateNoteContents" from the other fragment, so then I guess my acutal question is what does the "title" parameter represent (like is it necessary? does this method name need to be refactored/merged?)

Reply:
yep, this method should be refactored to "updateInode" or something along those lines since that's the primary function of this method now. the aim was to minimize the amount of data being passed between fragments while still being useful